samples/msal-react-samples/react-router-sample/README.md

```diff
@@ -17,6 +17,21 @@ This sample was bootstrapped with [Create React App](https://github.com/facebook
 1. `./src/ui-components/SignOutButton.jsx` - Example of how to get the `PublicClientApplication` instance using the `useMsal` hook and invoking a logout function.
 1. `./src/utils/MsGraphApiCall.js` - Example of how to call the MS Graph API with an access token.
 
+### (Optional) MSAL React and class components
+
+For a demonstration of how to use MSAL React with class components, see: `./src/pages/ProfileClassComponent.jsx`.
+
+*After* you initialize `MsalProvider`, there are 3 approaches you can take to protect your class components with MSAL React:
+
+1. Wrap each component that you want to protect with `withMsal` higher-order component (HOC) (see [Profile](./src/pages/ProfileClassComponent.jsx#Profile))
+1. Pass down the `msalContext` as prop to a child component from a parent component that is wrapped with `withMsal` HOC. (see [ProfileContent](./src/pages/ProfileClassComponent.jsx#ProfileContent))
```

Comment:
Not sure if this description is specifically referencing what is demonstrated in this sample or is speaking more generally. In a general sense when using this approach it doesn't matter how the parent component obtained the context it's passing down to the child. It could have been wrapped with `withMsal` or obtained the context from the raw context or the parent could be a function component that got the context using the `useMsal` hook.

I would either remove the part about the parent being wrapped with `withMsal` or say something like "from a parent component that has access to the `msalContext` via one of the other means"

Reply:
Sounds good! Went with "from a parent component that has access to the msalContext via one of the other means"